app/controllers/admin/oral_history_access_requests_controller.rb

```diff
@@ -1,12 +1,28 @@
 # STAFF FACING
 # Public facing actions are in app/controllers/oral_history_access_requests_controller.rb
 require 'csv'
+require "active_support/all"
```

Comment:
You should not need to require `active_support/all` here, as we're in a Rails app, it should already be there I think?

Curious why you added this; if it's actually needed, let's documetn why? (And it makes me worry what's going wrong or what I don't understand if it's actually needed!)